velox/vector/tests/VectorMakerTest.cpp

```diff
@@ -381,6 +381,83 @@ TEST_F(VectorMakerTest, arrayVectorWithNulls) {
   }
 }
 
+TEST_F(VectorMakerTest, arrayVectorFromJson) {
+  auto arrayVector = maker_.arrayVectorFromJson<int32_t>({
+      "null",
+      "[]",
+      "[1, 2, 3]",
+      "[1, 2, null, 4]",
+      "[null, null]",
+  });
+
```

Comment:
alternatively you can test it with an array created from the other API and call testing::assertEqua..